### DISCUSSION TITLE
SCP not responding

### DISCUSSION THREAD

**Discussion by Kavyamb** [semiautomaticgit/SemiAutomaticClassificationPlugin#325] [General]
As I try to create a band set of set of rasters, the process of bandset creation suddenly stopped with a message - The Semi-Automatic Classification Plugin is not responding. I'm currently using the QGIS 3.32.1 and SCP version 8 and the issue is arising since there was update to the SCP plugin to version 8.

**Comment by semiautomaticgit** [comment]
Hello @Kavyamb , what images are you using?
Also, please copy the log file following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#how-can-i-report-an-error
